xmake/scripts/xrepo/envs/ndk.lua

```diff
@@ -0,0 +1 @@
+add_requires("ndk")
```

Comment:
Some environments are not necessary built-in, users can add their own configuration according to their needs. https://xrepo.xmake.io/#/getting_started?id=manage-virtual-environments

We will only consider some of the more commonly used toolchain environments that are built in.

for example. emscripten, zig, ...

Reply:
so please remove it.